python/grass/jupyter/__init__.py

```diff
@@ -50,6 +50,12 @@
 
 >>> gj.init("grassdata/nc_basic_spm_grass7/user1")
 
+.. note::
+    Contrary to typical command line / GUI module usage, grass.jupyter
+    enables output overwrite by default. The default command line behaviour can be
+    restored by setting GRASS_OVERWRITE environmental variable to "0" after
+    gj.init() call: `os.environ["GRASS_OVERWRITE"] = "0"`
```

Comment:
```suggestion
    `gj.init()` call: `os.environ["GRASS_OVERWRITE"] = "0"`.
```

In general, it would be nice to have something like `gj.init(..., overwrite=False)` to avoid `os.environ["GRASS_OVERWRITE"] = "0"`, but we need to add couple lines to handle that.

Reply:
There is a rationale for defaulting to overwrite and I assume most of time such behavior is the right one. As long as we have documented this quirk and listed a workaround, I don't see much to gain form complicating the code.
This was just a surprise for me as I wrote code I expected to fail but it didn't.